### PR TITLE
Customers

### PR DESCRIPTION
**Pull request by k-rahman**
Complete registration form.

### CODE REVIEW

**Comment by t9haja00** [comment]
Yeah.  Very good. 
Maybe we could have a small comment for each field in the form.

File: application/views/users/register.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        <div class="card card-body col-md-8 bg-light shadow-lg  mt-5"><!-- Card Start -->
+            <form class="form-row" action="" method=""><!-- Form Start -->
+
+                <h2 class="form-text col-12 text-center mb-5">Create an account</h2>
+                <p class="form-text col-12 text-danger font-italic">
+                    <small>Required fields are marked by
+                        <strong><abbr title="required"> (*) </abbr></strong>
+                    </small>
+                </p>
+
```
Comment (t9haja00) [inline application/views/users/register.php]: First name

File: application/views/users/register.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    <label class="form-control-label sr-only" for="fname">First name</label>
+                    <div class="input-group">
+                        <input class="form-control" type="text" id="fname" placeholder="First name"
+                                name="firstname" required>
+                        <div class="input-group-append">
+                            <div class="input-group-text bg-white text-danger">*</div>
+                        </div>
+                    </div><!-- Input Group -->
+                </div><!-- Form Group -->
+
```
Comment (t9haja00) [inline application/views/users/register.php]: Last name

Comment (t9haja00) [reply application/views/users/register.php]: and etc..?

Comment (k-rahman) [reply application/views/users/register.php]: Sure no problem, i will include these comments in the next commit.

File: application/views/users/register.php (hunk 1 above)
Comment (k-rahman) [reply application/views/users/register.php]: Sure no problem, will be included in next commits.

**Pull request merged**